src/Raven.Studio/typescript/viewmodels/shell.ts

```diff
@@ -197,9 +197,18 @@ class shell extends viewModelBase {
         this.onBootstrapFinishedTask
             .done(() => {
                 registration.showRegistrationDialogIfNeeded(license.licenseStatus());
+                this.tryReopen();
             });
     }
 
+    private tryReopen() {
+        const random = Math.random() * (3 - 1) + 1;
```

Comment:
3-1 = 2 Is it not substracted for purpose or bug?

Reply:
was used for testing :)